roles/download/defaults/main.yml

```diff
@@ -517,24 +517,24 @@ cni_binary_checksums:
 
 calicoctl_binary_checksums:
   arm:
-    v3.23.1: 0
+    v3.23.2: 0
     v3.22.3: 0
     v3.21.5: 0
   amd64:
-    v3.23.1: e8fd04d776df5571917512560800bf77f3cdf36ca864c9cae966cb74d62ba4fe
+    v3.23.2: e8fd04d776df5571917512560800bf77f3cdf36ca864c9cae966cb74d62ba4fe
```

Comment:
You forgot to change the checksum here
> $ sha256sum /mnt/c/Users/E026692/Downloads/calicoctl-linux-amd64
3784200cdfc0106c9987df2048d219bb91147f0cc3fa365b36279ac82ea37c7a  /mnt/c/Users/E026692/Downloads/calicoctl-linux-amd64

Reply:
Sorry missed that here, just pushed it.